target/linux/qualcommax/ipq807x/base-files/etc/hotplug.d/ieee80211/11_fix_wifi_mac

```diff
@@ -30,4 +30,10 @@ case "$board" in
 		[ "$PHYNBR" = "1" ] && macaddr_add $label_mac 1 > /sys${DEVPATH}/macaddress
 		[ "$PHYNBR" = "2" ] && macaddr_add $label_mac 3 > /sys${DEVPATH}/macaddress
 		;;
+	zyxel,nbg7815)
+		label_mac=$(fw_printenv | awk -F'=' '/ethaddr/ {print $2}')
```

Comment:
The label's MAC address is already defined in dts. To get the MAC address, you can use the `get_mac_label` function.

Reply:
Thx for this hint. I've changed it accordingly.